### DISCUSSION TITLE
Add automatic subtitles to YouTube editable

### DISCUSSION THREAD

**Discussion by YoungBlood143** [pimcore/pimcore#12351] [Q&A / Support]
Hi everyone,
I use the Video Editable with YouTube as the source and would like to include automatic subtitles. I tried adding "&cc_lang_pref=es&cc_load_policy=1" to the ID inside the video edit dialog, but that doesn't work. Is there any way to add that?
Thanks in advance!

**Comment by dvesh3** [comment]
please try using the embed URL e.g. `/embed/9_epxjgipZo?cc_load_policy=1&hl=en&cc_lang_pref=en`
